ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh

```diff
@@ -16,6 +16,23 @@ tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted.tar.gz"
 # Prepare configuration and run
 scp "${SSHOPTS[@]}" "${CLUSTER_PROFILE_DIR}/pull-secret" "root@${IP}:pull-secret"
 
+if [ "${ENVIRONMENT:-}" != "local" ]; then
```

Comment:
Why default to empty? Let it fail on unset

Reply:
good idea

Reply:
done